service/http.proxy/pom.xml

```diff
@@ -7,6 +6,7 @@
         <groupId>org.kaazing</groupId>
         <artifactId>gateway</artifactId>
         <version>develop-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
```

Comment:
So, it doesn't reference parent, i.e. service/pom.xml ? Do we even have that one ?